api/client/gro_client.py

```diff
@@ -224,6 +230,38 @@ def get_data_points(self, **selections):
         # Return data points in input units if not unit is specified
         return data_points
 
+    def GDH(self, gdh_selection):
```

Comment:
@nemozen @JGrinst   https://github.com/gro-intelligence/api-client/pull/168 added support to optional params e.g unit_id, show_revisions etc

Reply:
looks good merged